content/post/3.1.0.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Release 3.1.0 is available
+date: 2024-06-29
+type: release
+linked: true
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+[Download](https://ratis.apache.org/downloads.html)
+
+This is a maintenance release with multiple improvements and bugfixes. See the [changes between 3.0.1 and 3.1.0](https://github.com/apache/ratis/compare/ratis-3.0.1...ratis-3.1.0) releases.
```

Comment:
@SzyWilliam , thanks again for working on the release! 

Similar to Apache Hadoop versioning (https://hadoop.apache.org/versioning.html)
```
<major>.<minor>.<maintenance>
```
3.1.0 is a `minor` release.  We may fix it in #1121 .   Sorry that I did not see this earlier.